apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -1,212 +1,247 @@
-      await page.locator('[placeholder="email\\@example\\.com"]').fill(`${memberUser.username}@example.com`);
-      await page.getByTestId("invite-new-member-button").click();
-      // wait for the second member to be added to the pending-member-list.
-      await page.getByTestId("pending-member-list").locator("li:nth-child(2)").waitFor();
-      await page.locator("[data-testid=publish-button]").click();
-      await expect(page).toHaveURL(/\/settings\/teams\/(\d+)\/event-type$/i);
-      // create a round-robin event type and finish
-      await expect(page.locator('button[value="ROUND_ROBIN"]')).toBeVisible();
-      await page.click('button[value="ROUND_ROBIN"]');
-      await page.fill("[name=title]", "roundRobin");
-      await page.getByTestId("finish-button").click();
-      await page.waitForURL(/\/settings\/teams\/(\d+)\/profile$/i);
-      // Going to create an managed event type
-      await page.goto("/event-types");
-      await page.getByTestId("new-event-type").click();
-      await page.getByTestId("option-team-1").click();
-      // Expecting we can add a managed event type as team owner
-      await expect(page.locator('button[value="MANAGED"]')).toBeVisible();
-
-      // Actually creating a managed event type to test things further
-      await page.click('button[value="MANAGED"]');
-      await page.fill("[name=title]", "managed");
-      await page.click("[type=submit]");
-
-      await page.waitForURL("event-types/**");
-    });
-
-    await test.step("Managed event type has unlocked fields for admin", async () => {
-      await page.getByTestId("vertical-tab-event_setup_tab_title").click();
-      await page.getByTestId("update-eventtype").waitFor();
-      await expect(page.locator('input[name="title"]')).toBeEditable();
-      await expect(page.locator('input[name="slug"]')).toBeEditable();
-      await expect(page.locator('input[name="length"]')).toBeEditable();
-      await adminUser.apiLogin();
-    });
-
-    await test.step("Managed event type exists for added member", async () => {
-      // Now we need to accept the invitation as member and come back in as admin to
-      // assign the member in the managed event type
-      await memberUser.apiLogin();
-
-      await page.goto("/teams");
-      await page.locator('button[data-testid^="accept-invitation"]').click();
-      await page.getByText("Member").waitFor();
-
-      await page.goto("/auth/logout");
-
-      // Coming back as team owner to assign member user to managed event
-      await adminUser.apiLogin();
-      await page.goto("/event-types");
-      await page.getByTestId("event-types").locator('a[title="managed"]').click();
-      await page.getByTestId("vertical-tab-assignment").click();
-      await page.getByTestId("assignment-dropdown").click();
-
-      await page.getByTestId(`select-option-${memberUser.id}`).click();
-      await page.locator('[type="submit"]').click();
-      await page.getByTestId("toast-success").waitFor();
```

Comment:
Now we wait for the POST request instead of relying on toast which is very flaky.